### PR TITLE
Changes to go live.

### PR DESCRIPTION
**Pull request by dkorolev**
Hi @mzhurovich ,

Minor cleanup in Current to make user-facing code more eye-pleasing. Also, a few bugfixes :-)

Thanks,
Dima

### CODE REVIEW

File: TypeSystem/Serialization/json.h [hunk 1]
```diff
@@ -290,6 +290,14 @@ struct LoadFromJSONImpl {
     destination = static_cast<T>(source.GetInt64());
   }
 
+  // `double`.
+  static void Load(rapidjson::Value& source, double& destination, const std::string& path) {
```
Comment (mzhurovich) [inline TypeSystem/Serialization/json.h]: What about `float`?

File: Bricks/util/random.h [hunk 2]
```diff
@@ -103,4 +103,14 @@ inline double RandomDouble(const double a, const double b) { return RandomReal<d
 }  // namespace random
 }  // namespace bricks
 
+using bricks::random::RandomInt;
+using bricks::random::RandomLong;
+using bricks::random::RandomInt64;
```
Comment (mzhurovich) [inline Bricks/util/random.h]: s/Int64/ULong/

File: TypeSystem/Serialization/json.h (hunk 1 above)
Comment (dkorolev) [inline TypeSystem/Serialization/json.h]: c2850c1 :-)

File: Bricks/util/random.h (hunk 2 above)
Comment (dkorolev) [inline Bricks/util/random.h]: Perhaps the other way around: retire `Int` and `Long` versions, leaving only the ones tied to byte-count-specific types?

Comment (mzhurovich) [inline Bricks/util/random.h]: Sounds reasonable :)

**Comment by mzhurovich** [comment]
@dkorolev LGTM!

**Pull request merged**